src/kernels/utf8.rs

```diff
@@ -75,6 +75,39 @@ pub fn add_utf8_arrays(lhs: &Utf8Array<i64>, rhs: &Utf8Array<i64>) -> DaftResult
     Ok(result)
 }
 
+#[allow(dead_code)] // TODO: Integrate and remove me!
+pub fn endswith_utf8_arrays(
```

Comment:
this has to be called by a DataArray which is then dispatched from a series call.

Reply:
> Currently unsure about the best way to link this to Series - maybe we don't need to register the function with Series itself, and can just call it from python/series.rs like so?
> 
> Python in series.py: Series.endswith(...)
> Rust in python/series.rs: endswith_utf8_arrays(self, other)

I wasn't sure if we wanted to add a method to Series that would call this method, or just dispatch from `python/series.rs` directly?